packages/webapp/utils/web3.ts

```diff
@@ -0,0 +1,13 @@
+import { fetchBalance } from "wagmi/actions"
+
+/**
+ * Fetch tokens for a wallet
+ * @param holder token holder address
+ * @param tokens list of tokens to find
+ * @returns the data of the tokens
+ */
+export function getTokensValues(holder: `0x${string}`, tokens: `0x${string}`[]) {
```

Comment:
not sure what the convention is throughout this project, but always nice to include the return type